docs/src/content/docs/guides/sidebar.mdx

```diff
@@ -34,14 +34,14 @@ The following sidebar will be automatically generated:
 <SidebarPreview
 	config={[
 		{
-			label: 'guides',
+			label: 'Guides',
```

Comment:
This is a 1:1 match of the `guides/` directory showed above (line 24) as we are in the case of a default sidebar which is entirely autogenerated.